WordPress/Classes/Models/PublicizeService.swift

```diff
@@ -41,11 +41,6 @@ public extension PublicizeService {
         case threads
         case unknown
 
-        /// Returns the local image for the icon representing the social network.
-        var localIconImage: UIImage {
-            WPStyleGuide.socialIcon(for: rawValue as NSString)
-        }
-
```

Comment:
`socialIcon(for service: NSString) -> UIImage` is defined in the apps target / Keystone, which WordPressData does not have access to. 

By moving it away from here, we unblock adding this required extension to the framework.